src/charm.py

```diff
@@ -496,6 +507,18 @@ def _update_endpoint_info(self) -> None:
             scope=" ".join(SUPPORTED_SCOPES),
         )
 
+    def _get_login_ui_endpoint_info(self, key: str) -> Optional[str]:
+        login_ui_url = ""
+        if self.model.relations[self._login_ui_relation_name]:
+            try:
+                login_ui_endpoints = self.login_ui_endpoints.get_login_ui_endpoints()
+                login_ui_url = login_ui_endpoints[key]
```

Comment:
Here we could simply `return login_ui_endpoints[key]` and not set the `login_ui_url` variable